build.gradle

```diff
@@ -307,7 +307,7 @@ task githubResources(type: ProcessResources) {
 		include '**'
 		version = project.property('version')
 		def channel = 'stable'
-		if (version.contains('pre'))
+		if (version.contains('pre') || version.contains('beta'))
```

Comment:
How come the checks here check against `pre` and `beta` individually, while the one in SkriptConfig checks if the version contains a hyphen

Reply:
Because i forgot to change that line too, thanks